File: 3Week/15Day/과제2/four.py
def solution(dot):
    if 0 < dot[0] and 0 < dot[1]:
        return 1
    elif 0 > dot[0] and 0 < dot[1]:
        return 2
    elif 0 > dot[0] and 0 > dot[1]:
        return 3
    elif 0 < dot[0] and 0 > dot[1]:
        return 4

File: 3Week/15Day/과제/four.java
import java.util.ArrayList;
import java.util.Arrays;

class four {
    public int solution(int[][] dots) {
        int width = 0;
        int height = 0;

        ArrayList<int[]> list = new ArrayList<>(Arrays.asList(dots));

        list.sort((i1, i2) -> {
            if(i1[0] == i2[0]){
                return i1[1] - i2[1];
            } 
            return i1[0] - i2[0];
        });
         
         
        width = list.get(2)[0] - list.get(1)[0];
        height = list.get(1)[1] - list.get(2)[1];

        return width * height;
    }
}
